util/gdas_init/run_v16.chgres.sh

```diff
@@ -59,13 +59,30 @@ rm -fr $WORKDIR
 mkdir -p $WORKDIR
 cd $WORKDIR
 
```

Comment:
@GeorgeGayno-NOAA Sounds a good plan. I will modify it as you suggested and test the code.

Reply:
@ChunxiZhang-NOAA I gave you some bad advice. Placing this new logic in the config file won't work for GDAS runs. Let me rework the logic and get back to you.

Reply:
@GeorgeGayno-NOAA OK, but I tested it worked for GDAS runs, not for the latest UFS_UTILS version though.

Reply:
@GeorgeGayno-NOAA My GDAS test run logs on Hera: /scratch1/NCEPDEV/hwrf/save/Chunxi.Zhang/Fixfv3/latest/UFS_UTILS/util/gdas_init

Reply:
The problem for GDAS runs is that the hires and enkf members can be at different resolutions. For example, the default resolutions are C192/C96. When I tested your branch, both the hires and enkf members used C192. I think the logic for setting the orog data will need to be moved to its own script and then "sourced" from each 'run' script. I moved the following logic to its own script:

```
if [ ${MEMBER} == 'gdas' ] || [ ${MEMBER} == 'gfs' ] || [ ${MEMBER} == 'hires' ] ; then
  CTAR=${CRES_HIRES}
else
  CTAR=${CRES_ENKF}
fi

if [ "${FRAC_ORO:-"no"}" = "yes" ]; then
  if  [ ${CTAR} == 'C48' ] ; then
    OCNRES='400'
  elif [ ${CTAR} == 'C96' ] ; then
    OCNRES='100'
  elif [ ${CTAR} == 'C192' ] ; then
    OCNRES='050'
  elif [ ${CTAR} == 'C384' ] || [ ${CTAR} == 'C768' ] || [ ${CTAR} == 'C1152' ]; then
    OCNRES='025'
  fi
  ORO_DIR="${CTAR}.mx${OCNRES}_frac"
  ORO_NAME="oro_${CTAR}.mx${OCNRES}"
else
  ORO_DIR="${CTAR}"
  ORO_NAME="${CTAR}_oro_data"
fi
```

Then 'sourced' it from one of the 'run' scripts as follows:

```
 mkdir -p $OUTDIR
 mkdir -p $OUTDIR/INPUT

+source $UFS_DIR/util/gdas_init/set_fixed_files.sh
+
 cat << EOF > fort.41
```

That seemed to work. You may have a better way.

Reply:
That's odd. My C192/C192 tests on both Hera and wcoss2 were successful. Which platform did you use?

Reply:
> That's odd. My C192/C192 tests on both Hera and wcoss2 were successful. Which platform did you use?

Try a C192/C96 test.

Reply:
C192/C96 were successful on both Hera and wcoss2.

Reply:
I am looking at your Hera test - /scratch1/NCEPDEV/hwrf/save/Chunxi.Zhang/Fixfv3/latest/UFS_UTILS/util/gdas_init. You specified a hires of C192, and enkf of C96. But if you look in log.001, chgres is reading the C192 fix files:

```
- INPUT DATA FROM FV3 GAUSSIAN NETCDF FILE.
 - OPEN TARGET GRID MOSAIC FILE:
 /scratch1/NCEPDEV/hwrf/save/Chunxi.Zhang/Fixfv3/latest/UFS_UTILS/util/gdas_init
 /../../fix/orog/C192.mx050_frac/C192_mosaic.nc
 - TRACER NAME IN OUTPUT FILE sphum
```

Reply:
@GeorgeGayno-NOAA I see. Because variable 'member' has values like 1, 2, 3.... in run_*.chgres.sh, it is impossible to use CDUMP in the config file to get the correct 'CTAR'.  I made changes and put the definition of 'CTAR' back to the run_*.chgres.sh files. Then source the set_fixed_files.sh in the run_*.chgres.sh. Now tests for C192/C96 passed.